op-challenger2/game/fault/solver/game_solver_test.go

```diff
@@ -208,98 +444,103 @@ func TestCalculateNextActions(t *testing.T) {
 			builder := claimBuilder.GameBuilder(faulttest.WithInvalidValue(!test.rootClaimCorrect))
 			test.setupGame(builder)
 			game := builder.Game
-
-			solver := NewGameSolver(maxDepth, trace.NewSimpleTraceAccessor(claimBuilder.CorrectTraceProvider()), types.CallDataType)
-			postState, actions := runStep(t, solver, game, claimBuilder.CorrectTraceProvider())
+			accessor := trace.NewSimpleTraceAccessor(claimBuilder.CorrectTraceProvider())
+			solver := NewGameSolver(maxDepth, accessor, types.CallDataType)
+			postState, actions := runStep2(t, solver, game, claimBuilder.CorrectTraceProvider())
+			preimage := getStepPreimage(t, builder, game, accessor)
```

Comment:
Better to move it to `ExpectStepV2` to make it more clear?

Reply:
The GetStepData2 function requires an accessor, but when test.setupGame is called, there is no such parameter.